python/cugraph/cugraph/dask/sampling/uniform_neighbor_sample.py

```diff
@@ -418,8 +418,9 @@ def uniform_neighbor_sample(
                 input_graph.renumber_map.renumbered_src_col_name
             ].dtype,
         )
-    elif with_edge_properties and batch_id_list is None:
-        batch_id_list = cudf.Series(cp.zeros(len(start_list), dtype="int32"))
+    elif with_edge_properties and with_batch_ids is False:
+        start_list = start_list.to_frame()
+        start_list[batch_id_n] = cudf.Series(cp.zeros(len(start_list), dtype="int32"))
```

Comment:
I'll update this with a function that generates the list of zeros locally instead of copying it.